app/markets/Info.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { shortenAddress } from '../../utils';
+import { Icon, IconButton } from '@swapr/ui';
+import { FixedProductMarketMaker } from '../../queries/omen';
+import { GNOSIS_SCAN_URL, KLEROS_URL, REALITY_URL } from '@/constants';
+
+export const Info = ({ market }: { market: FixedProductMarketMaker }) => {
+  const [clipboardIcon, setClipboardIcon] = useState<'copy' | 'tick'>('copy');
+  const id = market.id;
+
+  return (
+    <>
+      <div className="flex items-center space-x-1 py-4">
+        <span>{shortenAddress(id)}</span>
+        <IconButton
+          onClick={() => {
+            navigator.clipboard.writeText(id);
+            setClipboardIcon('tick');
+
+            setTimeout(() => {
+              setClipboardIcon('copy');
+            }, 3000);
+          }}
+          variant="ghost"
+          name={clipboardIcon}
+          size="sm"
+        />
+      </div>
+      <a
+        href={`${GNOSIS_SCAN_URL}/address/${id}`}
+        target="_blank"
+        className="flex items-center space-x-1 py-4"
+      >
+        <span>View market contract</span>
+        <Icon name="arrow-top-right" size={16} />
+      </a>
+      {market.question && (
+        <a
+          href={`${REALITY_URL}/app/#!/network/100/question/0x79e32ae03fb27b07c89c0c568f80287c01ca2e57-${market.question.id}`}
```

Comment:
this url has some hardcoded values, I think it's better to extract it to a variable.

Reply:
It's the same string value that we have in the Bet component too.